docs/sources/project/find-an-issue.md

```diff
@@ -195,12 +195,12 @@ To sync your repository:
           (use "git push" to publish your local commits)
         nothing to commit, working directory clean
 
-    Your local repository now has any changes from the `upstream` remote.  You
+    Your local repository now has many changes from the `upstream` remote.  You
```

Comment:
I don't think this is always likely to be the case.  Since this guide is intended to be worked through end-to-end in a single or very short time spam, it is more likely there are few or any.